src/NHibernate.Test/Async/NHSpecificTest/NH1316/Fixture.cs

```diff
@@ -9,10 +9,8 @@
 
 
 using System;
-using System.Data.Common;
-using NUnit.Framework;
 using NHibernate.Dialect;
-
+using NUnit.Framework;
```

Comment:
On my first approval I have not actually verified this change is legit. I guessed this is a "fix by the way" of a previous PR lacking a small bit of async regen tests. Is that it, or some unexpected change instead?

Reply:
Yes it is.